src/page/bell/Bell.ts

```diff
@@ -347,26 +334,26 @@ export default class Bell {
 
         const permission =
           await OneSignal._context._permissionManager.getPermissionStatus();
-        let bellState: BellState;
+        let bellState: BellStateType;
         if (isSubscribed.current.optedIn) {
-          bellState = Bell.STATES.SUBSCRIBED;
+          bellState = BellState._Subscribed;
         } else if (permission === 'denied') {
-          bellState = Bell.STATES.BLOCKED;
+          bellState = BellState._Blocked;
         } else {
-          bellState = Bell.STATES.UNSUBSCRIBED;
+          bellState = BellState._Unsubscribed;
         }
         this.setState(bellState, this._ignoreSubscriptionState);
       },
     );
 
-    OneSignal._emitter.on(Bell.EVENTS.STATE_CHANGED, (state) => {
+    OneSignal._emitter.on(BellEvent._StateChanged, (state) => {
       if (!this.launcher.element) {
         // Notify button doesn't exist
         return;
       }
-      if (state.to === Bell.STATES.SUBSCRIBED) {
+      if (state.to === BellState._Subscribed) {
         this.launcher.inactivate();
-      } else if (state.to === Bell.STATES.UNSUBSCRIBED || Bell.STATES.BLOCKED) {
+      } else if (state.to === BellState._Unsubscribed || BellState._Blocked) {
```

Comment:
Doesn't `BellState._Blocked` always evaluate to truthy? I'm not sure what this condition is trying to check for.